CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ Versioning](https://semver.org/spec/v2.0.0.html).
 - Push attendance when a student is on stage
 - Add Legacy LTI configuration XML
 - Add an instructor widget to modify join the discusion mode
+- Add a forced join mode which automatically joins students to the discussion
 
```

Comment:
in the commit message :    

When join mode is forced, users automatically ~~joins~~ -> join the live stage, so
no users would be able to populate the other participants list.